README.md

```diff
@@ -1,6 +1,6 @@
 # A-Puzzle-A-Day Solver
 
-A solver of [DragonFjord](https://www.dragonfjord.com/)'s [A-Puzzle-A-Day](https://www.dragonfjord.com/product/a-puzzle-a-day/) and similar puzzle(s).
+A solver of [DragonFjord](https://www.dragonfjord.com/)'s [A-Puzzle-A-Day](https://www.dragonfjord.com/product/a-puzzle-a-day/) and similar puzzle(s) like [this one with tetrominoes](https://puzzleparadise.net/listing/puzzle-calendar-solve-for-each-day-of-the-year-cherry-pieces-and-walnut-border/107535).
```

Comment:
At https://github.com/keiichiw/a-puzzle-a-day-solver/issues/4, an author of A-Puzzle-A-Day asked us to remove links to similar puzzles' site. In my understanding, they would be happy for me to implement a similar puzzle solver, but would not be very happy to have the link in the documentation or web page. 
So could you please remove this link, too?
It's kind of a gentlemen's agreement, but I'd like to pay tribute to the authors of the original A-Puzzle-A-Day puzzle.

Reply:
Sure thing! My only concern is ensuring that the actual puzzle can be described unambiguously. To that end, I've taken pictures of the puzzle I own, scaled them down, and added these to the repo. That way there's no external link.